packages/instant-apps-components/src/components/instant-apps-create/instant-apps-create.test.ts

```diff
@@ -0,0 +1,123 @@
+import { expect, test, describe } from 'vitest';
+
+import '../../../dist/components/instant-apps-create.js';
+//import './instant-apps-create.js';
+// import MapView  from 'esri/views/MapView' ;
+import Portal from "@arcgis/core/portal/Portal.js";
+import WebScene from '@arcgis/core/WebScene';
+import config from '@arcgis/core/config'
+// import SceneView from '@arcgis/core/views/SceneView';
+
+
+describe('inline', async () =>{
+    const create = document.createElement('instant-apps-create');
+    create.setAttribute('mode','inline');
+    create.setAttribute('show-scale-bar','true');
+    create.setAttribute('id','create-inline');
+    test('test if create option map works in inline', async () => {
+
+        config.portalUrl = 'https://nw-brews.mapsdevext.arcgis.com';
+        const portal = new Portal({
```

Comment:
you don't need to instantiate a portal if you already set `esri/config.portalUrl`. You can access the portal like so: `esri/WebScene.portalItem.portal`